loader/src/injector/hook.cpp

```diff
@@ -587,18 +606,19 @@ void ZygiskContext::fork_pre() {
     */
     sigmask(SIG_BLOCK, SIGCHLD);
     pid = old_fork();
-    if (pid != 0 || flags[SKIP_FD_SANITIZATION])
+    if (pid != 0 || flags[SKIP_FD_SANITIZATION]) {
         return;
+    }
 
     /* INFO: Record all open fds */
-    DIR *dir = opendir("/proc/self/fd");
-    if (dir == nullptr) {
-        PLOGE("Failed to open /proc/self/fd");
+    DIR *dir = opendir(DIR_FD);
+    if (dir == NULL) {
+        PLOGE("Failed to open %s", DIR_FD);
 
         return;
     }
 
-    struct dirent *entry;
+    struct dirent const* entry;
```

Comment:
```suggestion
    struct dirent const *entry;
```

Though I feel this is unnecessary since the compiler can figure stuff like this.